src/effects/effectmanifest.h

```diff
@@ -82,6 +83,14 @@ class EffectManifest final {
         m_isMixingEQ = value;
     }
 
+    virtual const bool& isWaveformChangingEQ() const {
+        return m_isWaveformChangingEQ;
```

Comment:
Ok, so no consensus.
If we have a EQ, that for example had a button that effects the high frequencies, it can have a property on this parameter, that it changes the high gain.
In this case we have only the seeping Bell, which effects the mid range. So we may decide for a knob that visualises it in our waveform EQ fake.